docs/src/pages/[platform]/getting-started/migration/migration.vue.mdx

```diff
@@ -150,3 +150,7 @@ please see the [Form Field Customization](../connected-components/authenticator/
 ### CSS Styling
 
 The 2.x version of the `Authenticator` has a completely different set of CSS variables. Please look over the [Amplify CSS Variables](../connected-components/authenticator/customization#styling) section for more information.
+
+### `onAuthUiStateChange`
```

Comment:
I don't think this applies to Vue?

Reply:
It would, because `onAuthUiStateChange` was a cross-framework feature in V1, right?